src/dtos/monitor-location-base.dto.ts

```diff
@@ -52,7 +59,14 @@ export class MonitorLocationBaseDTO {
   @IsOptional()
   @IsIsoFormat({
     message: (args: ValidationArguments) => {
-      return `${args.property} [MONLOC-FATAL-A] The value : ${args.value} for ${args.property} must be a valid ISO date format yyyy-mm-dd`;
+      return `The value of [${args.value}] for [${args.property}] must be a valid ISO date format [YYYY-MM-DD] for [${KEY}].`;
+    },
+  })
+  @IsValidDate({
```

Comment:
Isn't this check redundant with @IsIsoFormat ?

Reply:
![image](https://github.com/US-EPA-CAMD/easey-monitor-plan-api/assets/102534033/9f0fa8a3-d895-440c-ac72-246ece839e6e)

@IsIsoFormat does not check for valid date.